src/bin/set-version/set_version.rs

```diff
@@ -90,6 +106,7 @@ fn exec(args: VersionArgs) -> CargoResult<()> {
         all,
         dry_run,
         workspace,
+        workspace_version,
```

Comment:
imo I'm not a fan of commits that make no-op user visible changes like this as seeing the two sides of the change are important.

(overall though, I really appreciate commit breakdown!)

Reply:
Merged with the commit that came after that added the functionnality